.idea/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Default ignored files
```

Comment:
All of these files in the .idea directory are just project settings for the IDE I'm using (Jetbrains CLion). They can be ignored.